circleci/pod/redis-cluster/Dockerfile

```diff
@@ -0,0 +1,39 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ARG REDIS_VERSION
+
+FROM redis:${REDIS_VERSION}-alpine
+
+RUN apk --update --no-cache add supervisor ruby ca-certificates openssl && \
+  gem install --no-document redis && \
+  REDIS_TRIB_PATH=/usr/local/bin/redis-trib && \
+  wget https://raw.githubusercontent.com/antirez/redis/${REDIS_VERSION}/src/redis-trib.rb -O $REDIS_TRIB_PATH && \
+  chmod +x $REDIS_TRIB_PATH
+
+COPY start-4.x.sh /start.sh
+
+RUN ["chmod", "+x", "/start.sh"]
+
+VOLUME /data
+WORKDIR /
+
+ENV CLUSTER_ANNOUNCE_IP 127.0.0.1
+
+EXPOSE 5000 5001 5002 5003 5004 5005
```

Comment:
We don't need to expose as it's exposed in the compose file?

Reply:
Yep, there is no need to expose those port in this scenario.